specification/e-referrals-service-api.yaml

```diff
@@ -3303,7 +3303,89 @@ paths:
         - bearerAuth: []
       description: |
         ## Overview
-         TBD
+        Use this endpoint to cancel a referral and the current appointment, if there is one. Cancelling a referral is a one-way process and results in an end-state. No further progression of care can occur once this action has been performed.
+
+        A reason for cancelling an appointment must be provided. A comment on the cancellation is conditionally mandatory depending on the reason selected. See pre-requisites and related referrals for more information.
+
+        You can check if you can cancel a referral by calling [[HYPERLINK_A029]].
+
+        e-RS does not allow a referral to be cancelled if there is a current appointment. Therefore, if there is a current appointment associated with the referral, this *must* be cancelled in advance, or at the same time.
```

Comment:
Should be re-worded. You can cancel a referral with an appointment.

The overview should state how the user can know what options they have, and also **how they can check if the referral has a current appointment**

Reply:
BAs agreed to change this to "e-RS does not allow a referral to be canceled, without also canceling the current appointment."